panel/widgets/chatbox.py

```diff
@@ -205,47 +222,79 @@ class ChatBox(CompositeWidget):
     >>> ChatBox(value=[{"You": "Hello!"}, {"Bot": ["How may I help?", "I'm a bot."]}])
     """
 
-    value = param.List(default=[], item_type=Dict, doc="""
+    value = param.List(
```

Comment:
I realize you probably use auto-formatting and this is annoying but at least for the time being I would like to keep param definitions consistent in their formatting.

Reply:
Yeah I do, but no worries!